ci-operator/config/openshift/cloud-provider-aws/openshift-cloud-provider-aws-master.yaml

```
@@ -49,6 +81,9 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+
+rpm_build_commands: yum-builddep -y ecr-credential-provider.spec && chmod +x openshift-hack/build-rpm.sh && openshift-hack/build-rpm.sh 4.16.0
```

Shouldn't we be using the `dnf` version of this?
```suggestion
rpm_build_commands: dnf builddep -y ecr-credential-provider.spec && chmod +x openshift-hack/build-rpm.sh && openshift-hack/build-rpm.sh 4.16.0
```

---

I'll change it - honestly not sure. Copy pasted from cri-o.